hello-world/src/workflows.ts

```diff
@@ -0,0 +1,20 @@
+// @@@SNIPSTART nodejs-hello-workflow
+import { configureActivities } from '@temporalio/workflow';
+
+// Only import the activity types
+import type * as activities from './activities';
+
+const { greet } = configureActivities<typeof activities>({
+  type: 'remote',
+  startToCloseTimeout: '30 minutes',
+});
+
+/** A workflow that simply calls an activity */
+export const ExampleWorkflow = (name: string) => {
+  return {
+    async execute(): Promise<string> {
+      return await greet(name);
+    },
+  };
+};
```

Comment:
Do we like this syntax?

```suggestion
export const ExampleWorkflow = (name: string) => ({
  async execute(): Promise<string> {
    return await greet(name);
  },
});
```

Reply:
I'm a big fan of this syntax too, more concise and less banana code.